src/HDWallet.cpp

```diff
@@ -252,6 +256,7 @@ HDNode getNode(const HDWallet& wallet, TWCurve curve, const DerivationPath& deri
     auto node = getMasterNode(wallet, curve);
     for (auto& index : derivationPath.indices) {
         switch (privateKeyType) {
+            case HDWallet::PrivateKeyTypeHD:
             case HDWallet::PrivateKeyTypeExtended96:
                 // special handling for extended
                 hdnode_private_ckd_cardano(&node, index.derivationIndex());
```

Comment:
Just to confirm, here hdnode_private_ckd_cardano() is the right method for PrivateKeyTypeHD?

Reply:
Yes. For the private key, the cardano function is the correct one. We did not need to make a new one for it.